.github/workflows/stale.yml

```diff
@@ -4,17 +4,23 @@ on:
   schedule:
     - cron: '0 0 * * *'
 
+permissions:
+  issues: write
+  pull-requests: write
+
 jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/stale@main
         id: stale
         with:
-          debug-only: true
-          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove the stale label or comment or this will be considered for closing.'
+          debug-only: false
+          ascending: true
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove the stale label or comment or this will be closed in 14 days from this message.'
           days-before-stale: 30
-          days-before-close: -1
+          days-before-close: 14
+          operations-per-run: 500
```

Comment:
ah interesting, this could use some love I guess: https://github.com/o1-labs/proof-systems/blob/master/.github/workflows/stale.yml